lib/cucumber/events/step_definition_loaded.rb

```diff
@@ -0,0 +1,20 @@
+module Cucumber
+  module Events
+
+    # Event fired after each step definition has been loaded
+    class StepDefinitionLoaded
```

Comment:
Considering that the event is fired in the method `register_rb_step_definition`, you are absolutely right.